side.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Introduction</title>
+<link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel=stylesheet href=se.css>
+<link rel=prev href=quick.html title="Quick Tour">
+<link rel=next href=install.html title=Install>
+<script src=docs.js></script>
+
+<h1>Selenium-IDE</h1>
+
+<h3>Introduction</h3>
+
+<p>The <em>Selenium-IDE(Integrated Development Environment) </em>is the tool you use to develop your
```

Comment:
You don't develop tests with IDE (Noooooooo!) It's a record/playback tool that is designed to offer the selenium API and to allow people to export to a programming language.